### PR TITLE
Correction Made

### PR DESCRIPTION
**Pull request by babsmatic1**
My correction is now made

### CODE REVIEW

**Comment by DeleMike** [comment]
> My correction is now made

Did you just work on the `main` branch??? 

I'm seeing you wanna merge from main to master? 

This is wrong sir. You need to merge from your own branch to the main. 

I ssid before, simply clone this repo. If you have do `git pull`. Then, checkout to a working branch where you have to complete your task and then push your work. 


Please only request to merge into main. Never code in the main branch. If you realise you have been working on the main, do not commit, stash your changes(untracked included) and go to a new branch. 

In conclusion, I might not be able to merge your work.

**Comment by DeleMike** [comment]
kindly check this out @ade555
